### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by DariaNastas**
[DEMO LINK](https://DariaNastas.github.io/react_person/)

### CODE REVIEW

**Comment by DariaNastas** [comment]
image
image

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
```
Comment (anastasiiavorobiova) [inline src/components/Person/Person.jsx:1]: Consider using destructuring here

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,22 @@
...
+  return (
+    <div>
+      <section className="Person">
+        <h2 className="Person__name">My name is {person.name}</h2>
```
Comment (anastasiiavorobiova) [inline src/components/Person/Person.jsx:5]:
```suggestion
        <h2 className="Person__name">My name is {name}</h2>
```

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,22 @@
...
+        {person.age ? <p className="Person__age">I am {person.age}</p> : null}
```
Comment (anastasiiavorobiova) [inline src/components/Person/Person.jsx:6]:
```suggestion
        {age ? <p className="Person__age">I am {age}</p> : null}
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

**Review by anastasiiavorobiova — APPROVED** [review]
Awesome!

**Pull request closed without merging**